src/main/java/org/broadinstitute/hellbender/utils/test/CommandLineProgramTestInterface.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import htsjdk.samtools.util.Log;
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.logging.BunnyLog;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility interface for CommandLine Program testing.
+ */
+public interface CommandLineProgramTestInterface {
```

Comment:
The name isn't great.  It's hard for me to think of a better one though.  `CommandLineProgramTester` is the obvious alternative, but it's not really very good either.  I'll think about it...